clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/provider/view/EcsClusterProvider.java

```diff
@@ -36,4 +51,52 @@ public EcsClusterProvider(Cache cacheView) {
   public Collection<EcsCluster> getAllEcsClusters() {
     return ecsClusterCacheClient.getAll();
   }
+
+  // TODO include[] input of Describe Cluster is not part of this implementation, need to implement
+  // in the future.
+  public Collection<Cluster> getEcsClusterDescriptions(String account, String region) {
+    Collection<Cluster> clusters = new ArrayList<>();
+    List<String> filteredEcsClusters =
+        ecsClusterCacheClient.getAll().stream()
+            .filter(
+                cluster ->
+                    cluster.getRegion().equals(region) && cluster.getAccount().equals(account))
```

Comment:
It could be possible for cluster.getRegion and getAccount to be null. in this case it would lead to potential NPEs. One way to avoid this is to reverse your comparison:

For instance, you could update the

Current: `cluster.getRegion().equals(region) && cluster.getAccount().equals(account))`

to 

Updated: `region.equals(cluster.getRegion()) && account.equals(cluster.getAccount())`

In this case, if cluster.getRegion or cluster.getAccount is null, this won't thrown an NPE.